meson.build

```diff
@@ -210,7 +232,7 @@ if get_option('tests')
       ipc_test_files = {
           'ipc-decoder': {
               'src': 'decoder',
-              'deps': [nanoarrow_ipc_dep, arrow_dep, gtest_dep, gmock_dep],
+              'deps': [nanoarrow_ipc_dep, flatcc_dep, arrow_dep, gtest_dep, gmock_dep],
```

Comment:
Nice catch on this - noticed in https://github.com/apache/arrow-nanoarrow/pull/673 as well